app/routes/git_tokens.py

```
@@ -175,22 +203,23 @@
-            return APIResponse.success(
-                message=constants.TOKEN_DELETED_SUCCESSFULLY
-            )
+
+        uuid_id = uuid.UUID(id)  # Ensure it's a valid UUID
+
+        find_token = "SELECT label, id FROM git_label WHERE id = $1"
+        res = await db_client.execute_query_one(find_token, (uuid_id,))
+
+        if res:
+
+            deleted_label = await db_client.delete_rows("git_label", "id = $1", (id,))
```

Ruff (0.11.9)

Local variable `deleted_label` is assigned to but never used

Remove assignment to unused variable `deleted_label`

(F841)